Workbooks/Performance/Single Page Application Insights/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Single Page Application Insights",
```

Comment:
You may want to put Mezzurite in the title or users without it will be confused to see no data in their non-Mezzurite SPA reports. The text in the details will likely be cropped due to its length.

Reply:
The report checks to see if there any mezzurite logs, if not we have a help section that is shown with links back to our github and other mezzurite assests.